### PR TITLE
[KED-2812] Add docs paragraph about CLI hooks

### PR DESCRIPTION
**Pull request by lorenabalan**
## Checklist

- [x] Read the [contributing](https://github.com/quantumblacklabs/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/quantumblacklabs/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by MerelTheisenQB — APPROVED** [review]
LGTM 👍

File: docs/source/07_extend_kedro/02_hooks.md [hunk 1]
```diff
@@ -57,6 +57,16 @@ In addition, Kedro defines Hook specifications to register certain library compo
 
 The naming convention for registration hooks is `register_<library_component>`.
 
+
+#### CLI hooks
+
+Lastly, Kedro defines a small set of CLI hooks, that inject additional behaviour around execution of a Kedro CLI command:
```
Comment (AntonyMilneQB) [inline docs/source/07_extend_kedro/02_hooks.md:63]:
```suggestion
Lastly, Kedro defines a small set of CLI hooks that inject additional behaviour around execution of a Kedro CLI command:
```

**Review by AntonyMilneQB — APPROVED** [review]
Good stuff! Do ignore my extremely pedantic comma removal if you like.

**Comment by deepyaman** [comment]
> Good stuff! Do ignore my extremely pedantic comma removal if you like.

+1 on the pedantic comma removal. I like these catches. :P

File: docs/source/07_extend_kedro/02_hooks.md [hunk 2]
```diff
@@ -57,6 +57,16 @@ In addition, Kedro defines Hook specifications to register certain library compo
...
+
+* `before_command_run`
+
+This is what the [`kedro-telemetry`](https://github.com/quantumblacklabs/kedro-telemetry) plugin relies on under the hood in order to be able to collect CLI usage statistics.
```
Comment (yetudada) [inline docs/source/07_extend_kedro/02_hooks.md:67]:
```suggestion
This is what the [`kedro-telemetry`](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-telemetry) plugin relies on under the hood in order to be able to collect CLI usage statistics.
```

**Review by yetudada — APPROVED** [review]
Approved - but just updated the URL for `kedro-telemetry`.

**Pull request closed without merging**